### ISSUE TITLE
other public datasets，overfitting issue

### ISSUE THREAD

**Issue by YHaooo-4508** [mks0601/3DMPPE_POSENET_RELEASE#135]
Hello, I am trying to adapt the MPII_3DHP dataset on your code,  but it's totally overfitting
First I train it on H36M+MPII，the MPJPE is about 50+. This is normal
Then i change the dataset using MPII_3DHP+MPII to train. 50MPJPE in trainset but 200+ in validate dataset.
Have you conducted experiments on MPII_3DHP dataset？ Why is there such a serious overfitting？

**Comment by mks0601** [comment]
I'm not clear about your experiments.
1. First I train it on H36M+MPII，the MPJPE is about 50+. This is normal -> what is the test set?
2. Then i change the dataset using MPII_3DHP+MPII to train. 50MPJPE in trainset but 200+ in validate dataset. -> what are the training and validation sets?

**Comment by YHaooo-4508** [comment]
> I'm not clear about your experiments.
> 
> 1. First I train it on H36M+MPII，the MPJPE is about 50+. This is normal -> what is the test set?
> 2. Then i change the dataset using MPII_3DHP+MPII to train. 50MPJPE in trainset but 200+ in validate dataset. -> what are the training and validation sets?

1、train with H36M + MPII2D train dataset；test with H36M validate dataset
2、train with MPII_3DHP+MPII2D train dataset；test with  MPII_3DHP validate dataset

**Comment by mks0601** [comment]
I don't think this is from overfitting. I guess there is something wrong in your evaluation code?

**Comment by YHaooo-4508** [comment]
> I don't think this is from overfitting. I guess there is something wrong in your evaluation code?

I checked the code and found that there was an issue with bbox, which has been fixed. However, the results of subsequent testing were still overfitting. The validation set MPJPE was around 150, trainset is about 60.  Here are some of my visualization results. Some samples also have poor results
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/80744649/5a4d68f0-9275-4c9a-b66f-5edb91b66df6)
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/80744649/3f54db7c-05f3-412c-b96f-657100bcf0e0)
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/80744649/c304021b-8652-48a8-a03d-a6fd3b578a06)
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/80744649/e9e4a930-7e34-4327-80dd-7c10335e6bb5)

**Comment by mks0601** [comment]
Error 60 on training set means definitely underfitting

**Comment by YHaooo-4508** [comment]
> Error 60 on training set means definitely underfitting

Firstly, thank you for communicating with me. 
I know that continuing training may improve performance in the training set, but at this point, the validation set error no longer decreases, so I stopped training, and there was a significant difference between the validation set error and the training set error. 
By the way, during my training and validation process, I did not use object detection algorithms to predict bbox, but instead used key point positions to fit the bounding box. I am not sure if this method will affect the training.
